ts/modules/dumbbell.src.ts

```diff
@@ -0,0 +1,567 @@
+            public options: DumbbellSeriesOptions;
+            public pointClass: typeof DumbbellPoint;
+            public points: Array<DumbbellPoint>;
+            public trackerGroups: Array<string>;
+            public crispCol: ColumnSeries['crispCol'];
+            public translatePoint: AreaRangeSeries['translate'];
+            public setShapeArgs: ColumnRangeSeries['translate'];
+            public seriesDrawPoints: AreaRangeSeries['drawPoints'];
+            public drawTracker: TrackerMixin['drawTrackerPoint'];
+            public drawGraph: any;
+            public crispConnector(points: SVGPathArray): SVGPathArray;
+            public getConnectorAttribs(point: DumbbellPoint): SVGAttributes;
+            public drawConnector(point: DumbbellPoint): void;
+            public getColumnMetrics(): ColumnMetricsObject;
+            public translate(): void;
+            public drawPoints(): void;
+            public markerAttribs(): SVGAttributes;
+            public pointAttribs(
+                point: DumbbellPoint,
+                state: string
+            ): SVGAttributes;
+        }
+    }
+}
+
+var pick = H.pick,
+    seriesType = H.seriesType,
+    seriesTypes = H.seriesTypes,
+    seriesProto = H.Series.prototype,
+    areaRangeProto = seriesTypes.arearange.prototype,
+    columnRangeProto = seriesTypes.columnrange.prototype,
+    colProto = seriesTypes.column.prototype,
+    areaRangePointProto = areaRangeProto.pointClass.prototype;
+/**
+ * The dumbbell series is a cartesian series with higher and lower values for
+ * each point along an X axis, connected with a line between the values.
+ * Requires `highcharts-more.js` and `modules/dumbbell.js`.
+ *
+ * @sample {highcharts} highcharts/demo/dumbbell/
+ *         Dumbbell chart
+ * @sample {highcharts} highcharts/series-dumbbell/styled-mode-dumbbell/
+ *         Styled mode
+ *
+ * @extends      plotOptions.arearange
+ * @product      highcharts highstock
+ * @excluding    fillColor, fillOpacity, lineWidth, stack, stacking,
+ *               stickyTracking, trackByArea
+ * @since        next
+ * @optionparent plotOptions.dumbbell
+ */
+seriesType<Highcharts.DumbbellSeries>('dumbbell', 'arearange', {
+    /** @ignore-option */
+    trackByArea: false,
+    /** @ignore-option */
+    fillColor: 'none',
+    /** @ignore-option */
+    lineWidth: 0,
+    pointRange: 1,
+    /**
+     * Pixel width of the line that connects dumbbell point's values.
+     *
+     * @since     next
+     * @product   highcharts highstock
+     */
+    connectorWidth: 1,
+    /** @ignore-option */
+    stickyTracking: false,
+    groupPadding: 0.2,
+    pointPadding: 0.1,
+    /**
+     * Color of the start markers in dumbbell graph.
+     *
+     * @type      {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+     * @since     next
+     * @product   highcharts highstock
+     */
+    startColor: '${palette.neutralColor80}',
```

Comment:
Since the this color is related to the `low` data property, shouldn't it be called `lowColor`?